libs/ui/src/ui_strings/election_strings.tsx

```diff
@@ -1,22 +1,104 @@
-import { Party } from '@votingworks/types';
+/* eslint-disable react/destructuring-assignment */
```

Comment:
Just curious, why is this necessary?

Reply:
Not sure why this happened, but once I switched over to using the `ElectionStringKey` enum as the keys, the eslint rule started interpreting the method params as `props` for components, so it complained about not destructuring `candidate` before referencing `candidate.id`, for example.

Reply:
Got it, thanks for explaining